vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -31,7 +31,7 @@ static YogaNodePtr make_yoga_node() {
 static int YogaLog(const YGConfigRef config, const YGNodeRef node, YGLogLevel level, const char *format, va_list args) {
   int len = _scprintf(format, args);
   std::string buffer(len + 1, '\0');
-  int c = vsnprintf_s(&buffer[0], len + 1, _TRUNCATE, format, args);
+  [[maybe_unused]] int c = vsnprintf_s(&buffer[0], len + 1, _TRUNCATE, format, args);
```

Comment:
>c [](start = 23, length = 1)

why not remove it?

Reply:
removed

---
In reply to: [370870518](https://github.com/microsoft/react-native-windows/pull/3958#discussion_r370870518) [](ancestors = 370870518)